ctera/src/main/java/ch/cyberduck/core/ctera/CteraListService.java

```diff
@@ -0,0 +1,55 @@
+package ch.cyberduck.core.ctera;
+
+/*
+ * Copyright (c) 2002-2024 iterate GmbH. All rights reserved.
+ * https://cyberduck.io/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+import ch.cyberduck.core.Path;
+import ch.cyberduck.core.dav.DAVListService;
+import ch.cyberduck.core.dav.DAVPathEncoder;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.github.sardine.DavResource;
+
+import static ch.cyberduck.core.ctera.CteraAclPermissionFeature.allCteraCustomACLQn;
+
+class CteraListService extends DAVListService {
+    private final CteraSession cteraSession;
+
+    public CteraListService(final CteraSession cteraSession) {
```

Comment:
Rename parameter to `session`.

Reply:
Done 4e7b873308259561fc41842e3d654bca436c71ee.